dnsapi/dns_freedns.sh

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env sh
+
+#This file name is "dns_freedns.sh"
+#So, here must be a method dns_freedns_add()
+#Which will be called by acme.sh to add the txt record to your api system.
+#returns 0 means success, otherwise error.
+#
+#Author: David Kerr
+#Report Bugs here: https://github.com/Neilpang/acme.sh
+#
+########  Public functions #####################
+
+# Export FreeDNS userid and password in folowing variables...
+#  FREEDNS_USER=username
```

Comment:
Reverted to all uppercase to solve Travis spellcheck failures.  As these are exported from user environment they should by convention be all uppercase.